mycroft/stt/__init__.py

```diff
@@ -122,13 +122,35 @@ def get_response(self, response):
             return None
 
 
+class BingSTT(TokenSTT):
+    def __init__(self, lang="en-us", config=None):
+        super(BingSTT, self).__init__("bing")
+
+    def execute(self, audio, language=None):
+        self.lang = language or self.lang
+        return self.recognizer.recognize_bing(audio, self.token,
+                                              self.lang)
+
+
+class HoundifySTT(BasicSTT):
+    def __init__(self, lang="en-us", config=None):
+        super(HoundifySTT, self).__init__("houndify")
+
+    def execute(self, audio, language=None):
+        self.lang = language or self.lang
+        return self.recognizer.recognize_houndify(audio, self.username,
+                                                  self.password, self.lang)
```

Comment:
Looking at the docs for houndify this doesn't seem quite right:

`recognize_houndify(self, audio_data, client_id, client_key, show_all=False)`

I think I would prefer `client_id` and `client_key` config parameters and not reuse username and password